ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -180,6 +180,15 @@ final class CommandsValidator(ledgerId: LedgerId, submissionIdGenerator: Submiss
       case ProtoEmpty =>
         Left(missingField("command"))
     }
+
+  private def extractOrGenerateSubmissionId(commands: ProtoCommands) = {
+    if (commands.submissionId.isEmpty) {
+      domain.SubmissionId(submissionIdGenerator.generate())
+    } else {
+      domain.SubmissionId(Ref.SubmissionId.assertFromString(commands.submissionId))
+    }
+  }
```

Comment:
I think you don't need the curly braces here.

Reply:
I usually prefer to always have them in cases like this. Is the convention to remove them in the codebase?

Reply:
LOL, "convention".

I think we _tend_ to use the curly braces when there are multiple lines or side effects, and avoid them otherwise. In addition, if one side of the `if` gets braces, the other side gets them too.

This is just anecdotal, though. I don't think there's much consistency.

Reply:
> I usually prefer to always have them in cases like this

What do they add?

> I think we tend to use the curly braces when there are multiple lines or side effects, and avoid them otherwise. In addition, if one side of the if gets braces, the other side gets them too.

My personal guideline is "all else being equal, less is better".

Reply:
> What do they add?

The reduce the chance that someone adds new code that he thinks it's inside the if block but because of the lack of braces it's actually outside the if block. 

> My personal guideline is "everything else being equal, less is better".

My main reason is that I prefer consistency, I know exactly what to expect and having all the if blocks with braces makes it easier to compute. 


Will remove the braces though as it's used in multiple places without it.

Reply:
Note that I was referring to the outer braces, not the ones of the `if...else` expression

Reply:
I agree that consistency of braces for `if...else` is useful and I think in the committer team we settled on rather having them.

Reply:
Ah, makes more sense now 👍 my brain fixated on the if. will change that

Reply:
(Although if you remove both the `if...else` braces and the outer braces, then there is no change one could inadvertently add code outside of the `if` nor `else` blocks)